administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,124 @@
+  width: inherit;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+type ApplicationVerificationProps = {
+  applicationVerificationAccessKey: string
+}
+
+type VerifiedState = 'verified' | 'rejected' | 'pending'
+
+const ApplicationVerification = ({ applicationVerificationAccessKey }: ApplicationVerificationProps) => {
+  const [verified, setVerifiedState] = useState<VerifiedState>('pending')
+  const config = useContext(ProjectConfigContext)
+  const appToaster = useAppToaster()
+  const showErrorToaster = () => appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+  const [verifyOrRejectApplicationVerification, { loading: verificationLoading }] =
+    useVerifyOrRejectApplicationVerificationMutation({
+      onError: error => {
+        console.error(error)
+        showErrorToaster()
+      },
+      onCompleted: ({ result }, clientOptions) => {
+        if (!result) {
+          console.error('Verify operation returned false.')
+          showErrorToaster()
+        } else {
+          if (!clientOptions?.variables?.verified) setVerifiedState('rejected')
+          else setVerifiedState('verified')
+        }
+      },
+    })
+
+  const submitApplicationVerification = (verified: boolean) => {
+    verifyOrRejectApplicationVerification({
+      variables: {
+        accessKey: applicationVerificationAccessKey,
+        verified,
+      },
+    })
+  }
+
+  const { loading, error, data, refetch } = useGetApplicationByApplicationVerificationAccessKeyQuery({
+    variables: { applicationVerificationAccessKey },
+  })
+
+  if (loading) return <Spinner />
+  else if (error || !data) return <ErrorHandler refetch={refetch} />
+  if (data.verification.rejectedDate || data.verification.verifiedDate)
+    return <CenteredMessage title='Dieser Antrag wurde bereits bearbeitet' />
+  if (verified !== 'pending')
+    return (
+      <CenteredMessage
+        title={`Der Antrag wurde Erfolgreich ${verified === 'verified' ? 'verifiziert' : 'abgelehnt'}`}
+        description='Sie können das Fenster jetzt schließen.'
+      />
+    )
+
+  const { jsonValue, id } = data.application
+  const jsonField = JSON.parse(jsonValue)
+  const baseUrl = `${process.env.REACT_APP_API_BASE_URL}/application/${config.projectId}/${id}`
+  return (
+    <Container>
+      <H1>{config.name}</H1>
+      <span>
+        Können Sie die Angaben, welche die Organisation <b>{data.verification.organizationName}</b> betreffen
+        bestätigen?
+      </span>
+      <ApplicationViewCard>
+        <JsonFieldView jsonField={jsonField} baseUrl={baseUrl} hierarchyIndex={0} />
+      </ApplicationViewCard>
+      <ButtonContainer>
+        <Button icon='cross' loading={verificationLoading} onClick={() => submitApplicationVerification(false)}>
+          Widersprechen
+        </Button>
+        <Button icon='tick' loading={verificationLoading} onClick={() => submitApplicationVerification(true)}>
+          Bestätigen
+        </Button>
```

Comment:
Maybe, we should add an info box that tells the user that this choice cannot be changed?